content/millennium/r4/entities/organization.md

```diff
@@ -95,6 +95,41 @@ List an individual Organization by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## $get-cg-for-mrcu
+
+Retrieve Caregiver Organizations that match the supplied care unit organization id:
+ 
+    GET /Organization/$get-cg-for-mrcu?_id=:id
```

Comment:
ok, check with @lalitha94 for sure but we had an offline conversation and I think we decided this needs to be search like this "GET /Organization/$get-cg-for-mrcu?:parameters".  That also makes me rethink the order and that we should put this operation as a section header between "Search" and "Retrieve by Id".

Reply:
Updated here: 36acecb3fbb535666f51aa41458f13d44716c6da

I was wondering though, do we need to have the parameters table documenting id (like general Search) or not have the parameters table to be consistent with Binary?

Reply:
I think because our example has :parameters, we'd need to add the parameters table documenting id like General Search.

Reply:
Updated here: ba3f00dafa17a9daa4b2a0b845caf48eeaec4312